pages/index.js

```diff
@@ -1,10 +1,71 @@
-import styled from "styled-components";
+import styled from 'styled-components';
+import { useState } from 'react';
 
 const Title = styled.h1`
   font-size: 50px;
   color: ${({ theme }) => theme.colors.primary};
 `;
 
 export default function Home() {
-  return <Title>My page</Title>;
+  const [meetings, setMeetings] = useState([]);
+
+  function handleSubmit(event) {
+    console.log(event);
+    event.preventDefault();
+
+    const titel = event.target.elements.titel.value;
+    const location = event.target.elements.location.value;
+    const activity = event.target.activities.value;
+
+    setMeetings(prevState => [...prevState, { titel, location, activity }]);
+
+    console.log(titel);
+    console.log(location);
+    console.log(activity);
+  }
+  return (
+    <>
+      <Title>Erstelle dein Treffpunkt</Title>
+      <form onSubmit={handleSubmit}>
+        <div>
+          <label>Titel</label>
+          <input type="text" name="titel" placeholder="Titel" required />
+        </div>
+        <div>
+          <label>Standorte</label>
+          <select name="location">
+            <option value="">Bitte auswählen</option>
+            <option value="stadtpark">Stadtpark</option>
+            <option value="elbe">Elbe</option>
+            <option value="alster">Alster</option>
+          </select>
+        </div>
+        <div>
+          <label>Aktivitäten</label>
+          <select name="activities">
+            <option value="">Bitte auswählen</option>
+            <option value="gassigehen">Gassi gehen</option>
+            <option value="sport">Sport</option>
+            <option value="otherthings">Sonstiges</option>
+          </select>
```

Comment:
Looking very good, I only would suggest to use the singular here, to stay consistent (you use location in the upper one and not locations)